test/react-web/client/graphql/queries.tsx

```diff
@@ -837,14 +854,15 @@ describe('queries', () => {
                 people: prev.allPeople.people.concat(fetchMoreResult.data.allPeople.people),
               },
             }),
-          });
-          // XXX add a test for the result here when #508 is merged and released
+          }).then(wrap(done, result => {
+            expect(result.data.allPeople.people).to.deep.equal(data1.allPeople.people);
+          }));
         } else if (count === 1) {
-          expect(props.data.variables).to.deep.equal(variables2);
+          expect(props.data.variables).to.deep.equal(variables);
```

Comment:
`fetchMore` is not going to change `data.variables` anymore, but I think this is more correct, no? (the variables for `fetchMore` might be completely different to those for the original query).